internal/plugins/ansible/plugin.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ansible
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+const pluginName = "ansible.sdk.operatorframework.io"
```

Comment:
```suggestion
const pluginName = "ansible" + plugins.DefaultNameQualifier
```

Reply:
This is fine, but I am surprised that the DefaultNameQualifier is `operator-sdk.io` given that we dont own that domain.

Reply:
^^ We should discuss this before 1.0

Reply:
https://github.com/operator-framework/operator-sdk/pull/3527